inst/css/filter-panel.css

```diff
@@ -156,6 +156,8 @@ a.remove_all:hover {
   animation: 0.75s ease-out 0s 1 shinyDataFilterEnlargeY,
     0.5s ease-in 0s 1 shinyDataFilterFadeIn;
   transform-origin: bottom;
+  width: 100%;
```

Comment:
I think we should probably revisit how we do this overlay as it feels very unstable @gogonzo

Reply:
For me bs4 image size is now correct in this case:

![image](https://user-images.githubusercontent.com/15201933/210835556-92b7501b-cb74-493a-b996-d6169452f606.png)

but bs3 is too big:
![image](https://user-images.githubusercontent.com/15201933/210835685-2adde4e1-a2d6-455b-8414-a5330ae26021.png)

as is bs5:
![image](https://user-images.githubusercontent.com/15201933/210835989-b7fe7fee-36e9-459d-a026-e6f672e86ba0.png)


so either we can leave it as the histogram is meant as a rough guide only so it's ok if it's not exactly right, or we should have a rethink

Reply:
That shift was caused by the margin setup here:

https://github.com/insightsengineering/teal.slice/blob/4de856b16662d8051e44ecd8d0fe24a2090f32f5/inst/css/filter-panel.css#L154

Not sure why we have the left and right margin but if I set to `auto` or `0px`, the div box width is correct.

```css
margin: 10px auto -25px;
```
or 
```css
margin: 10px 0px -25px;
```

Tested against all bs version, the range graphs and div boxes look correct.

Reply:
so without margin although the divs line up, the image doesn't line up with the range of the slider:

No margin
![image](https://user-images.githubusercontent.com/15201933/210962086-183d880a-f775-4852-b472-aace9eab61e6.png)

margin:
![image](https://user-images.githubusercontent.com/15201933/210963042-972d1322-4d1b-4c3a-b275-4afc63e49acb.png)


I'll see if there's a better way of matching them up...

Reply:
ok so I fixed this by on these slider (and only these ones) removing the extra bits of the slider beyond the range:
![image](https://user-images.githubusercontent.com/15201933/210970000-8a2d4574-fe60-4d7a-91ae-f4b2fd101724.png)

This doesn't affect the users ability to choose their range